src/style.css

```
@@ -1,3 +1,69 @@
+:root {
+  --link-color: #00ACDC;
```

I'd recommend you to change this naming a little. Since you can use this value not only for links
```suggestion
  --accent-color: #00ACDC;
```